dagshub/data_engine/model/query_result.py

```diff
@@ -876,6 +839,13 @@ def to_voxel51_dataset(self, **kwargs) -> "fo.Dataset":
         ds.merge_samples(samples)
         return ds
 
+    @staticmethod
+    def _get_predict_dict(predictions, remote_path, log_to_field):
+        return {
+            log_to_field: json.dumps(predictions[remote_path][0]).encode("utf-8"),
+            f"{log_to_field}_score": (0.0 if len(predictions[remote_path]) == 1 else predictions[remote_path][1]),
```

Comment:
Why set the score at 0.0?
Wouldn't it be better to not have it in the dict at all? I think a 0 confidence might be confusing

Reply:
Yeah makes sense, will fix